Mathlib/MeasureTheory/Measure/Haar/Basic.lean

```diff
@@ -590,33 +590,51 @@ inner regular Haar measures (and in particular for any Haar measure on a second
 
 open Pointwise
 
-/-- **Steinhaus Theorem** In any locally compact group `G` with an inner regular Haar measure `μ`,
+@[to_additive]
+private lemma steinhaus_mul_aux (μ : Measure G) [IsHaarMeasure μ] [μ.InnerRegularCompactLTTop]
+    [LocallyCompactSpace G] (E : Set G) (hE : MeasurableSet E)
+    (hEapprox : ∃ K ⊆ E, IsCompact K ∧ 0 < μ K) : E / E ∈ 𝓝 (1 : G) := by
+  /- For any inner regular measure `μ` and set `E` of positive measure, we can find a compact
+  set `K` of positive measure inside `E`. Further, there exists a neighborhood `V` of the
```

Comment:
Can you tweak the comment to the current situation?

Reply:
What about this?